centreon/src/Centreon/Domain/Log/Logger.php

```diff
@@ -32,6 +32,8 @@
  */
 class Logger implements LoggerInterface
 {
+    public const LEVEL_OFF = 900;
+
```

Comment:
⚠️ After looking at monolog, this is bad idea to use a non-existent level because it can raises Exceptions.

The static method [Logger::toMonologLevel()](https://github.com/Seldaek/monolog/blob/main/src/Monolog/Logger.php#L468-L500) is used by several handlers and raises an `InvalidArgumentException` when the integer is not one of the list of [Level](https://github.com/Seldaek/monolog/blob/main/src/Monolog/Level.php) enum.

Reply:
I suggets using the `EMERGENCY` level which is `600`

Reply:
handled in https://github.com/centreon/centreon/pull/1469